.github/ISSUE_TEMPLATE/avm_ci_environment_issue.yml

```diff
@@ -0,0 +1,44 @@
+name: AVM - CI Environment Issue 🚀
+description: Have an issue, question or suggestion for the CI Environment? Let us know!
```

Comment:
@eriqua, do you think it also makes sense to call out `utilities` seperately, or should we consider the 'CI Environment' as an overarching term for all things non-module-related?

Reply:
Nothing against being more specific and mentiong "CI Environment or utilities" in the issue description, similarly to what already done in PR templates

Reply:
@AlexanderSehr, @eriqua, 
- would you only change the description to: "Have an issue, question or suggestion for the CI Environment or its utilities? Let us know!"
- OR
- would you also change the template's name to "AVM - CI Environment or Utility Issue 🚀" AND the title to "[AVM CI Environment or Utility Issue]: "

My 2 cents: now that we have the concept of "utility modules" mentioning utilities in this context might be misleading (though we all understand that it's not the same thing).

Reply:
I prefer the former as it does avoid confusion.

If we were to go with the latter, then I'd suggest to call it something like `CI-Utility`.

Reply:
Good point Mate let's not mix them up. +1 for the former

Reply:
Lgtm. I'll leave @AlexanderSehr the honors as he opened this thread.